packages/elements-dev-portal/src/containers/StoplightProject.tsx

```diff
@@ -56,6 +62,10 @@ const StoplightProjectImpl: React.FC<StoplightProjectProps> = ({ projectId, hide
   let elem: JSX.Element;
   if (isLoadingNode || !isTocFetched) {
     elem = <Loading />;
+  } else if (isError) {
+    if ((nodeError as any).status === 402) {
+      elem = <UpgradeToStarter />;
+    }
```

Comment:
Why do we need to cast it as `any`? Is it an unknown? Can we maybe cast it to the runtime type? 

Also I know this is more of a question of API design, but is this really a good use case for 402? MDN lists it as "Experimental technology" and "nonstandard", and also it's a bit misleading: It's not the client (that's fetching the data) that needs to make a payment but the host itself. @philsturgeon I'd love to hear your thoughts on this and on 402 in general.